### PR TITLE
Feat/leaderboard

### PR DESCRIPTION
**Pull request by MRowan121**
1. What's this PR do?
- This PR creates the leaderboard component and connects it to the modal component.
- The leaderboard component has been set up with a fetch requests so that the modal is populated with all scores.
2. Where should the reviewer start?
- Leaderboard.js
- Modal.js
3. How should this be manually tested?
- npm start
4. Any background context you want to provide?
5. What are the relevant tickets?
6. Screenshots (if appropriate)
Screenshot 2023-04-04 at 4 17 42 PM
7. Questions:

### CODE REVIEW

File: src/Components/Leaderboard/Leaderboard.css
Comment (awirth224) [inline src/Components/Leaderboard/Leaderboard.css:1]: there is an extra space on line 42

File: src/Components/Leaderboard/Leaderboard.js
Comment (awirth224) [inline src/Components/Leaderboard/Leaderboard.js:1]: Let's remove the console log on line 31.

File: src/Components/Modal/Modal.js
Comment (awirth224) [inline src/Components/Modal/Modal.js:1]: Do we intend to use the commented out code later? (lines 12-17)

**Review by awirth224 — APPROVED** [review]

File: src/Components/Leaderboard/Leaderboard.css
Comment (MRowan121) [reply src/Components/Leaderboard/Leaderboard.css:1]: Just removed it

**Review by BobAiken — APPROVED** [review]
Good job separating the leaderboard into its own component.

**Pull request merged**